omigami/plotting.py

```diff
@@ -0,0 +1,153 @@
+from typing import List
+
+from PIL.PngImagePlugin import PngImageFile
+import pandas as pd
+from rdkit import Chem
+from rdkit.Chem import Draw
+from rdkit.Chem.rdchem import Mol
+import itertools
+import matplotlib.pyplot as plt
+import requests
+
+from omigami.config import CLASSYFIRE_URL, NPCLASSIFER_URL
+
+
+class MandatoryColumnMissingError(Exception):
+    pass
+
+
+# TODO: Dont know about the name though
```

Comment:
Maybe call it `<something>Plotter` instead of `<something>Plots`

Reply:
Not to confuse with Harry Plotter